### ISSUE TITLE
[BUG] - Issue after installing nextui component

### ISSUE THREAD

**Issue by Gitstar-OC** [nextui-org/nextui#3940]
### NextUI Version

2.4

### Describe the bug

The website is stopping directly after installing the nextui component 

```bash
⨯ ./app/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[14].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[14].oneOf[12].use[3]!./app/globals.css
SyntaxError: Unexpected token, expected "," (10:7)
Import trace for requested module:
./app/globals.css.webpack[javascript/auto]!=!./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[14].oneOf[12].use[2]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[14].oneOf[12].use[3]!./app/globals.css
./app/globals.css
 GET /_next/static/webpack/47d80cc62d2786fa.webpack.hot-update.json 500 in 5534ms
 GET /_next/static/webpack/47d80cc62d2786fa.webpack.hot-update.json 500 in 5534ms
 GET / 500 in 5594ms
 GET /_next/static/webpack/47d80cc62d2786fa.webpack.hot-update.json 500 in 5537ms
 ⚠ Fast Refresh had to perform a full reload. Read more: https://nextjs.org/docs/messages/fast-refresh-reload
 ⚠ Fast Refresh had to perform a full reload. Read more: https://nextjs.org/docs/messages/fast-refresh-reload
 ⚠ Fast Refresh had to perform a full reload due to a runtime error.       
 GET / 500 in 38ms
```

This is coming every time and I have tried this 4 times till now. 

### Steps to Reproduce the Bug or Issue

Installing nextui component is throwing this issue, this might be the reason anyone can reproduce it. I don't know if it can be reproduced or not or is it due to nextjs. 

### Expected behavior

It should not throw any error and the build should be successful. 

### Screenshots or Videos

![{713A27EF-EC0C-42D0-96E7-D2A64D3F3E14}](https://github.com/user-attachments/assets/f06bb9f4-8118-476d-a961-df57deb5e689)

### Operating System Version

Windows

### Browser

Chrome

**Comment by wingkwong** [comment]
please share a minimal reproducible environment.

**Comment by Gitstar-OC** [comment]
Alright, I will push this at a new github repository.

**Comment by Gitstar-OC** [comment]
Kindly clone [this repository](https://github.com/Gitstar-OC/PromptAThon)

Please note that
- I have removed nextui packages from it but I think you can still do the same thing, I am not going to run all of this commands again as it will take more time. 

Run this after cloning the repository 
`cd client`
`npm install`
`npm run dev`

You can now see that this runs and works properly. 

Now try to install any nextui component, I prefer tooltip
`npx nextui-cli@latest add tooltip` 

- I have previously added various components in it which worked, they can be located in the package.lock.json file as I have removed them from the package.json and installed the node_modules agian. 

After installing tooltip try to use it in the Navbar, or replace it like this 

```jsx
"use client";
import { useTheme } from "next-themes";
import { SunIcon, MoonIcon } from "@radix-ui/react-icons";
import {Tooltip} from "@nextui-org/tooltip";

export default function Navbar() {
  const { theme, setTheme } = useTheme();
  const isDarkMode = theme === "dark";

  return (
    <nav className="fixed max-w-xl w-full rounded-full mx-auto top-4 left-0 right-0 z-50 bg-white/80 dark:bg-black/80 text-black dark:text-white p-2 flex items-center justify-between shadow-md">
      <h1 className="text-xl font-mono">Prompt A Thon</h1>      
        <Tooltip content="Change theme">
            <button
              onClick={() => setTheme(isDarkMode ? "light" : "dark")}
              className="p-1 mr-1  rounded-lg transition-colors hover:bg-gray-200 dark:hover:bg-neutral-800"
            >
              {isDarkMode ? (
                <SunIcon className="w-6 h-6 text-white" />
              ) : (
                <MoonIcon className="w-6 h-6 text-gray-800" />
              )}
            </button>
        </Tooltip>
    </nav>
  );
}
```

This will throw an error similar to above!! Hope this helps, kindly tell me if you can fix this as the nextui tooltip is better than the shadcn one I am using currently.

**Comment by wingkwong** [comment]
preliminary findings:
- cli breaks tailwind config

**Comment by IsDyh01** [comment]
@wingkwong  Can i work on this？

**Comment by Gitstar-OC** [comment]
@IsDyh01 Sure, you can create a pull request to resolve this and suggest me how will it work.

**Comment by wingkwong** [comment]
@IsDyh01 this has been assigned. please check with other issues.

**Comment by winchesHe** [comment]
@Gitstar-OC you can change the content `"./node_modules/@nextui-org/theme/dist/components/[object Object].js"` to `'./node_modules/@nextui-org/theme/dist/**/*.{js,ts,jsx,tsx}'` in `client/tailwind.config.js` file for temporary repair

**Comment by Gitstar-OC** [comment]
Thanks for the quick fix!!

**Comment by winchesHe** [comment]
will be fixed in next nextui-cli v0.3.5 release

**Comment by winchesHe** [comment]
@Gitstar-OC Fixed you can run `npx nextui-cli@latest` or `npm i -g nextui-cli` to use latest cli
